pints/_optimisers/_irpropmin.py

```diff
@@ -127,12 +166,35 @@ def needs_sensitivities(self):
 
     def n_hyper_parameters(self):
         """ See :meth:`pints.TunableMethod.n_hyper_parameters()`. """
-        return 0
+        return 2
 
     def running(self):
         """ See :meth:`Optimiser.running()`. """
         return self._running
 
+    def set_hyper_parameters(self, x):
+        """
+        See :meth:`pints.TunableMethod.set_hyper_parameters()`.
+
+        The hyper-parameter vector is ``[min_step_size, max_step_size]``.
+        """
+        self.set_min_step_size(x[0])
+        self.set_max_step_size(x[1])
+
+    def set_max_step_size(self, step_size):
+        """
+        Sets the maximum step size (use ``None`` to let step sizes grow
+        indefinitely).
+        """
+        self._step_max = None if step_size is None else float(step_size)
+
+    def set_min_step_size(self, step_size):
+        """
+        Sets the minimum step size (use ``None`` to let step sizes shrink
+        indefinitely).
+        """
+        self._step_min = None if step_size is None else float(step_size)
```

Comment:
Same here. Checking step_max > step_min is probably a good idea, and that step_min > 0